qgroundcontrol.pro

```diff
@@ -266,6 +266,7 @@ HEADERS += \
     src/HomePositionManager.h \
     src/Joystick/Joystick.h \
     src/Joystick/JoystickManager.h \
+    src/VehicleSetup/JoystickConfigController.h \
```

Comment:
This add joystick support to all builds which will include iOS which is incorrect. Joystick support should now be everything but ios build.